stl/inc/tuple

```diff
@@ -882,14 +912,9 @@ struct _View_as_tuple<pair<_Ty1, _Ty2>> { // view a pair as a tuple
     using type = tuple<_Ty1, _Ty2>;
 };
 
-template <class _Ty, class... _Types>
-struct _View_as_tuple<array<_Ty, 0>, _Types...> { // view an array as a tuple; ends recursion at 0
-    using type = tuple<_Types...>;
-};
-
-template <class _Ty, size_t _Size, class... _Types>
-struct _View_as_tuple<array<_Ty, _Size>, _Types...>
-    : _View_as_tuple<array<_Ty, _Size - 1>, _Ty, _Types...> { // view an array as a tuple; counts down to 0
+template <class _Ty, size_t _Size>
+struct _View_as_tuple<array<_Ty, _Size>> { // view an array as a tuple
+    using type = typename _Repeat_type_in_tuple<_Ty, make_index_sequence<_Size>>::type;

```

Comment:
This is `using type = _Meta_repeat_n_c<_Size, _Ty, tuple>;` with the definition of `_Meta_repeat_n_c` from `<variant>` which we could push up into `<type_traits>`.

Reply:
Done